hyperpackage/hyperpackage/utilities.py

```diff
@@ -80,19 +88,17 @@ def write_hyperpack_to_s3(
         raise
 
     print(
-        "*** COMPLETED: The {} hyperpack was written to the {} S3 bucket ***".format(
-            s3_object_key, s3_bucket
+        f"*** COMPLETED: The {s3_object_key} hyperpack was written to the {s3_bucket} S3 bucket ***"
         )
-    )
 
 
 def write_json(dictionary, json_file_path):
     """Writes object to JSON format
     Args:
-        dictionary: python dict to be written to JSON
+        dictionary: pythson dict to be written to JSON
```

Comment:
Is "pythson" a portmanteau of "python" and "json"?

Reply:
how'd you knOW!? ;)